PayloadPkg/FirmwareUpdate/FirmwareUpdateHelper.c

```diff
@@ -390,6 +393,28 @@ UpdateNonRedundantComp (
   return Status;
 }
 
+/**
+  This function would check if update is for container Image.
+
+  @param[in]  Signature   Container signature
+
+  @retval  BOOLEAN        Container True/False
+
+**/
+BOOLEAN
+IsLzHeader (
+  IN  UINT32   Signature
+  )
+{
+
+  if ((Signature == LZ4_SIGNATURE) || (Signature == LZDM_SIGNATURE)

```

Comment:
Use IS_COMPRESSED macro instead.

Reply:
Please see definition in  BootloaderCommonPkg\Include\Library\DecompressLib.h

Reply:
Sure, I would use existing macro IS_COMPRESSED.